### ISSUE TITLE
Option to disable creation of objects that cannot be tagged

### ISSUE THREAD

**Issue by jagoodhand** [jagoodhand/terraform-snowflake-playground#14]
Users of the module should have an option to disable the user creation of objects that cannot be currently cleaned up by automation.

**Comment by pyule23** [comment]
Hi @jagoodhand - would an option here be to drop objects based on their create date alone rather than not allowing them? Also if you could marry up objects that cannot be tagged with the objects that can where there is a dependency you could piggyback on the tagged object dates.

**Comment by jagoodhand** [comment]
Yes, I like that. Will likely have an array as a choice option to control behaviour:

- block creation of non-taggable objects
- only allow non-taggable objects to exist for specified max object period
- new tag on taggable objects which pairs to a non-taggable object (will probably pull this out as a separate issue though as it is a lot more work)
